guides/common/attributes.adoc

```diff
@@ -96,7 +96,7 @@ ifeval::["{build}" == "foreman"]
 :ansible-namespace: `theforeman.foreman`
 :ansible-namespace-example: `theforeman.foreman._module_name_`
 :awx: AWX
-:BaseURL: https://docs.theforeman.org/master/
+:BaseURL: https://docs.theforeman.org/nightly/
```

Comment:
Please do the same on line 242 for the foreman-deb build

Reply:
Oh right, will change.